gitgud/levels/reworking/_cherrypick/test.spec

```diff
@@ -0,0 +1,11 @@
+1
+2 : 1
+3 (bugFix)
+4 : 1
+5 (side)
+6 : 1
+7 (another)
+8 : 1
```

Comment:
This challenge is going to require us to work on #43. Right now, the changes that you're trying to track here are just going to be looked at as new commits, so you'll need to list the commits as 3':1, 4', 7', per the instructions you listed

Take a look at #43 and explore the references to see the other issues that are involved here.